velox/type/Filter.cpp

```diff
@@ -118,11 +118,12 @@ BigintValuesUsingHashTable::BigintValuesUsingHashTable(
     bool nullAllowed)
     : Filter(true, nullAllowed, FilterKind::kBigintValuesUsingHashTable),
       min_(min),
-      max_(max) {
+      max_(max),
+      values_(values) {
   VELOX_CHECK(min < max, "min must be less than max");
   VELOX_CHECK(values.size() > 1, "values must contain at least 2 entries");
 
-  auto size = 1u << (uint32_t)std::log2(values.size() * 3);
+  auto size = 1u << (uint32_t)std::log2(values.size() * 5);
```

Comment:
What's the motivation for changing the constant here from 3 to 5? Would you add a comment in the code to explain how / why this constant is chosen?